From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@deneha__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

andynava committed:
Include Andre's commit

paulathurman committed:
edit pass: data-factory-batch1

paulathurman committed:
edit pass: data-factory-batch2

Commit:
add Azure to ttile

Commit:
update path to fix warning

ggailey777 committed:
Merge branch 'master' of github.com:MicrosoftDocs/azure-docs-pr into current

ggailey777 committed:
File rename per direction from Cosmos DB folks

ggailey777 committed:
fixed bug

ggailey777 committed:
Fixed the entry point description

paulathurman committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into data-factory-batch2

ggailey777 committed:
Merge branch 'patch-8' of github.com:ggailey777/azure-docs-pr into current

ggailey777 committed:
Initial updates to the VS quickstart

Commit:
update portal to lowercase

Commit:
update description

Commit:
update Azure Powershell link

paulathurman committed:
edit pass: data-factory-batch2

ggailey777 committed:
Merge branch 'master' of github.com:MicrosoftDocs/azure-docs-pr into current

ggailey777 committed:
VS tools cleanup items

ggailey777 committed:
Move local VS fixes

ggailey777 committed:
Tweak on versions

ggailey777 committed:
Merge branch 'master' of github.com:MicrosoftDocs/azure-docs-pr into current

ggailey777 committed:
Fixed image

ggailey777 committed:
Fix remaining images

carolinacmoravia committed:
M19319: Unclear term: "Job W"

]Hello, @georgewallace, 

Translator has reported possible source content issue.  We cannot understand the meaning of "Job W". Is this a typo? 

Please, help to check my comment tag added into the article and reply with explanation if source text should be fixed, or it is by design. If fix is needed, please, let me know either if you would like me to fix it in this PR, or if you will fix it in another PR. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR. 

Many thanks in advance.

carolinacmoravia committed:
Update automation-verify-runas-authentication.md

itechedit committed:
edit pass: mysql/howto-migrate-online

paulathurman committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into data-factory-batch2

paulathurman committed:
edit pass: data-factory-batch2

paulathurman committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into data-factory-batch1

ggailey777 committed:
Tom comments

paulathurman committed:
edit pass: data-factory-batch1

Ja-Dunn committed:
Merge pull request #31126 from v-savila/1157293

update Logic Apps LP

v-geberr committed:
appInsights

v-geberr committed:
appInsights

v-alje committed:
Bringing even with master.

ggailey777 committed:
Tom comments

ggailey777 committed:
Link fixes

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into release-mvc-logic-apps

v-geberr committed:
fn app insights

v-alje committed:
Bringing even with master.

v-geberr committed:
rough finished

v-geberr committed:
rough finished

v-geberr committed:
getting rid of IDs

v-geberr committed:
acrolinx

v-geberr committed:
project.json

v-geberr committed:
manage-keys

v-geberr committed:
numbering

ggailey777 committed:
Merge branch 'release-serverless-interactive' of github.com:MicrosoftDocs/azure-docs-pr into current

v-geberr committed:
fix

ecfan committed:
Add overview content

ecfan committed:
Merge pull request #6 from ecfan/working-mvc-logic-apps-overview

Add overview content

ecfan committed:
Add quickstart content

ecfan committed:
Merge pull request #7 from ecfan/working-mvc-logic-apps-quickstart

Add quickstart content

v-alje committed:
Bringing even with master.

ecfan committed:
Add 3rd tutorial

ecfan committed:
Merge pull request #8 from ecfan/working-mvc-logic-apps-tutorial3

Add 3rd tutorial

ecfan committed:
Add other tutorials

v-geberr committed:
edits

v-geberr committed:
fixes

v-geberr committed:
image reference

v-geberr committed:
fix

v-geberr committed:
fix

v-geberr committed:
fix

ggailey777 committed:
Updating the publishing part of the VS tutorial

ggailey777 committed:
Storage account creation info note

ggailey777 committed:
Fix warning

v-geberr committed:
TOC

v-geberr committed:
not node

v-geberr committed:
C#

ggailey777 committed:
Removed Note formatting

tdykstra committed:
request size and url length limits

v-geberr committed:
landing page for tutorials

v-geberr committed:
change text

v-alje committed:
Bringing even with master.

Ja-Dunn committed:
Merge pull request #31447 from ecfan/release-mvc-logic-apps

MVC Logic Apps - Add overview, quickstart, and 3rd tutorial

ecfan committed:
Merge branch 'working-mvc-logic-apps-tutorials' of https://github.com/ecfan/azure-docs-pr into working-mvc-logic-apps-tutorials

ecfan committed:
Merge pull request #9 from ecfan/working-mvc-logic-apps-tutorials

Add other tutorials

ecfan committed:
Fix Acrolinx issues

ecfan committed:
Fix Acrolinx

ecfan committed:
Merge pull request #10 from ecfan/working-mvc-logic-apps-tutorials

Fix Acrolinx

ecfan committed:
Replace TOC.md with TOC.yml

ktoliver committed:
Merge pull request #31532 from ecfan/release-mvc-logic-apps

MVC Logic Apps - Add more tutorials

ecfan committed:
Merge branch 'release-mvc-logic-apps' of https://github.com/Microsoft/azure-docs-pr into working-mvc-logic-apps-update-toc

ecfan committed:
Merge pull request #11 from ecfan/working-mvc-logic-apps-update-toc

Replace TOC.md with TOC.yml

ktoliver committed:
Merge pull request #31545 from ecfan/release-mvc-logic-apps

MVC Logic Apps: Replace TOC.md with updated TOC.yml

v-alje committed:
Bringing even with master.

ggailey777 committed:
Re-added Note formatting

v-alje committed:
Bringing even with master.

WilliamAntonRohm committed:
adding content and image files from PR 29332

WilliamAntonRohm committed:
adding TOCs

v-geberr committed:
c# switch to nodejs

v-geberr committed:
acrolinx and unused images

v-geberr committed:
link fix

v-geberr committed:
tiny edits

v-geberr committed:
next step

v-geberr committed:
fix

ecfan committed:
Update redirects

ecfan committed:
Merge pull request #12 from ecfan/mvc-redirect-updates

MVC Logic Apps - Update master redirection file

v-geberr committed:
fix

v-geberr committed:
rebuilt

ecfan committed:
Add missing bookmark

ecfan committed:
Merge pull request #13 from ecfan/mvc-redirect-updates

Add missing bookmark

v-geberr committed:
touch

ecfan committed:
Remove obsolete file

ecfan committed:
Merge pull request #14 from ecfan/mvc-redirect-updates

Delete "logic-apps-what-are-logic-apps.md"

ecfan committed:
Add bookmark

ecfan committed:
Restore file and add bookmark

ecfan committed:
Merge pull request #15 from ecfan/mvc-redirect-updates

Restore file and add bookmark

ecfan committed:
Update links

ecfan committed:
Merge pull request #16 from ecfan/mvc-redirect-updates

Update links

ecfan committed:
Update links

ecfan committed:
Merge pull request #17 from ecfan/mvc-redirect-updates

Update links

ecfan committed:
Update links and delete obsolete files

ecfan committed:
Merge pull request #18 from ecfan/mvc-redirect-updates

Update links and delete obsolete files

v-alje committed:
Bringing even with master.

v-geberr committed:
changes requested by Kamran

Juliako committed:
big updates

ggailey777 committed:
Application settings info

ggailey777 committed:
Acrolinx

ggailey777 committed:
fix H2

v-thepet committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into linkfix

yujhongmicrosoft committed:
Add files via upload

yujhongmicrosoft committed:
Add files via upload

ggailey777 committed:
Prevent H2 wrap

yujhongmicrosoft committed:
Update documentation-government-get-started-connect-with-vsts.md

yujhongmicrosoft committed:
Delete documentation-government-vsts-img11.png

yujhongmicrosoft committed:
Delete documentation-government-vsts-img13.png

yujhongmicrosoft committed:
Delete documentation-government-vsts-img15.png

carolinacmoravia committed:
C10685: Show code for image 

Hello, @BryanLa, 

Localization team has reported source content issue that causes localized version to have broken format compared to en-us version.
We need to fix the incorrect strucuture. 

Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR.

Many thanks in advance.

yujhongmicrosoft committed:
Delete documentation-government-vsts-img14.png

yujhongmicrosoft committed:
Delete documentation-government-vsts-img18.png

yujhongmicrosoft committed:
Delete documentation-government-vsts-img9.png

yujhongmicrosoft committed:
Add files via upload

carolinacmoravia committed:
Update active-directory-devhowto-adal-error-handling.md

ecfan committed:
Merge branch 'release-mvc-logic-apps' of https://github.com/MicrosoftDocs/azure-docs-pr into release-mvc-logic-apps

carolinacmoravia committed:
C9718: Missing space in INCLUDE structure

Hello, @BryanLa,

Localization team has reported source content issue that causes localized version to have broken format compared to en-us version. 

Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR.

Many thanks in advance.

ecfan committed:
Update links and change author on topic to delete

ecfan committed:
Remove dupe content

ecfan committed:
Remove obsolete file

mattbriggs committed:
adding stat stop azure stack topic

mattbriggs committed:
update link and text

tfitzmac committed:
revised publish marketplace

v-alje committed:
Bringing even with master.

Juliako committed:
Update api-management-howto-aad.md

tfitzmac committed:
reformatted

Lisaco88 committed:
Merge pull request #31618 from ecfan/release-mvc-logic-apps

MVC Logic Apps: Update master redirection file, update links, and remove obsolete topics

tfitzmac committed:
added registration detail

Juliako committed:
Update api-management-howto-aad.md

ecfan committed:
Fix How to section header in ToC

kgremban committed:
Added IoT Edge to breadcrump file

v-thepet committed:
link fixes

barbkess committed:
distributed table updates

v-thepet committed:
fix merge conflicts

barbkess committed:
fix the date

yujhongmicrosoft committed:
Update documentation-government-get-started-connect-with-vsts.md

v-thepet committed:
fix

yujhongmicrosoft committed:
Update documentation-government-get-started-connect-with-vsts.md

v-thepet committed:
more fixes

WilliamAntonRohm committed:
addressed all review comments

spelluru committed:
V2 - stored procedure to run SSIS package

v-thepet committed:
fix

v-geberr committed:
new UX

vladvino committed:
Update import-and-publish.md

v-geberr committed:
fix images

WilliamAntonRohm committed:
addressed non-blocking review comments from PR 31043

v-geberr committed:
fix images

sethmanheim committed:
Fixing minor typo

v-thepet committed:
fix

v-geberr committed:
azure branding

rjagiewich committed:
Merge pull request #31687 from tfitzmac/0117marketplace

revised publish marketplace

GitHubber17 committed:
Merge pull request #31599 from WilliamAntonRohm/W-hdi-01-29332-Nitin

3 more articles from Dev Guide for master (replaces PR 29332)

PRMerger19 committed:
Merge pull request #31720 from sethmanheim/typo-fix

Fixing minor typo

PRMerger18 committed:
Merge pull request #31721 from WilliamAntonRohm/W-hdi-01-18-speculative

fixed remaining two review comments from merged PR 31043

PRMerger15 committed:
Merge pull request #31717 from vladvino/patch-4

Minor corrections

kgremban committed:
Update toc.yml

v-thepet committed:
fix

Juliako committed:
Update api-management-howto-aad.md

Juliako committed:
Update api-management-howto-create-groups.md

Lisaco88 committed:
Merge pull request #31708 from barbkess/hash2

distributed table updates

rjagiewich committed:
Merge pull request #31722 from v-geberr/0118-entities-ux

0118 entities ux

GitHubber17 committed:
Merge pull request #31697 from kgremban/breadcrumb

Added IoT Edge to breadcrump file

v-thepet committed:
fix?

v-alje committed:
Bringing even with master.

Commit:
edit/remove note on datetimev2 locales

v-thepet committed:
last ones?

v-geberr committed:
new ux

Lisaco88 committed:
Merge pull request #31695 from ecfan/release-mvc-logic-apps

Fix How to section header in ToC

PRMerger14 committed:
Merge pull request #31731 from DeniseMak/prebuilt-1-18

edit/remove note on datetimev2 locales

jonbeck7 committed:
Update virtual-machines-common-sizes-compute.md

v-alje committed:
Merge pull request #31361 from MicrosoftDocs/release-mvc-logic-apps

Release mvc logic apps

PRMerger12 committed:
Merge pull request #31734 from jonbeck7/patch-15

Update virtual-machines-common-sizes-compute.md

spelluru committed:
Reviewed and updated

spelluru committed:
content freshness review

spelluru committed:
content freshness review

spelluru committed:
content freshness review

juhacket committed:
Update service-fabric-application-arm-resource.md
- Fixed swagger spec link
- Improved description of serviceName parameter

PRMerger16 committed:
Merge pull request #31738 from spelluru/adcfreshness0118

content freshness review

ktoliver committed:
Merge pull request #31650 from Juliako/APIMUpdates

big updates

PRMerger18 committed:
Merge pull request #31739 from spelluru/adfcfreshness0118-2

content freshness review

Commit:
Merge pull request #31740 from spelluru/adfcfreshness0118-3

content freshness review

ktoliver committed:
Merge pull request #31706 from v-thepet/linkfix

Linkfixes per work item https://mseng.visualstudio.com/TechnicalContent/_workitems/edit/11

ktoliver committed:
Merge pull request #31712 from spelluru/ssis0118-2

V2 - stored procedure to run SSIS package

ktoliver committed:
Merge pull request #31730 from v-geberr/0119-entity-ux

new ux

tamram committed:
fixing duplicate entry in TOC for blobs

linda33wj committed:
Add loopup activity size limit

linda33wj committed:
Tweak copy overview wording

linda33wj committed:
Reformat key info

PRMerger16 committed:
Merge pull request #31743 from linda33wj/master

Update ADF content

pareshmu committed:
Update er locations

Commit:
Merge pull request #31745 from pareshmu/patch-4

Update er locations

PRMerger19 committed:
Merge pull request #31293 from carolinacmoravia/patch-545

M19319: Unclear term: "Job W"

PRMerger19 committed:
Merge pull request #31670 from carolinacmoravia/patch-548

C10685: Show code for image

mumian committed:
review the Kafka getting started article

itechedit committed:
Merge pull request #31197 from ggailey777/current

Updates to the VS quickstart

Commit:
Merge pull request #31685 from carolinacmoravia/patch-549

C9718: Missing space in INCLUDE structure

v-geberr committed:
new UX

spelluru committed:
Maximum parallel executions per node

ktoliver committed:
Merge pull request #31681 from mattbriggs/mdb_startupandshutdown

adding stat stop azure stack topic

itechedit committed:
Merge pull request #31742 from tamram/tamram-0118

fixing duplicate entry in TOC for blobs

ktoliver committed:
Merge pull request #31746 from mumian/0119-kafka

review the Kafka getting started article

ktoliver committed:
Merge pull request #31747 from v-geberr/0119-tut-app-prog-nodejs

new UX

itechedit committed:
Merge pull request #31666 from yujhongmicrosoft/master

VSTS quickstart

spelluru committed:
minor edit

v-geberr committed:
old images that needed to be removed

ktoliver committed:
Merge pull request #31360 from v-geberr/0111-azFnAppInsights

0111 az fn app insights

PRMerger15 committed:
Merge pull request #31748 from spelluru/ssis0119

Maximum parallel executions per node

v-geberr committed:
adding BF blog filtered to LUIS category as resource

v-geberr committed:
adding BF blog filtered to LUIS category as resource

ShawnJackson committed:
Merge pull request #31310 from itechedit/mysql/howto-migrate-online

edit pass: mysql/howto-migrate-online

itechedit committed:
Merge pull request #31749 from v-geberr/0119-remaining-images-new-ux

old images that needed to be removed

Commit:
Update manager metadata

spelluru committed:
new include for connectors without Copy Data tool

ShawnJackson committed:
Update data-factory-spark.md

Commit:
Merge pull request #31753 from JennieHubbard/patch-101

Update manager metadata

v-alje committed:
Resolving merge conflict

Commit:
Merge pull request #31741 from juhacket/master

Update service-fabric-application-arm-resource.md

ktoliver committed:
Merge pull request #31752 from v-geberr/0119-bf-blog-resource

0119 bf blog resource

ShawnJackson committed:
Merge pull request #31334 from paulathurman/data-factory-batch1

Edit pass: Data factory batch1

ShawnJackson committed:
Update data-factory-data-processing-using-batch.md

ktoliver committed:
Merge pull request #31754 from spelluru/connectorinclude0119

new include for connectors without Copy Data tool

v-alje committed:
Merge pull request #31756 from MicrosoftDocs/FromPublicRepo

From public repo

Commit:
Merge pull request #31506 from tdykstra/patch-8

request size and url length limits

ShawnJackson committed:
Merge pull request #31338 from paulathurman/data-factory-batch2

edit pass: Data factory batch2

aashishb committed:
Corrected a step for local operationalization

rwike77 committed:
freshness

rwike77 committed:
freshness

Commit:
Merge pull request #31759 from rwike77/patch-14

freshness

Commit:
Merge pull request #31762 from rwike77/patch-15

freshness

PRMerger20 committed:
Merge pull request #31758 from aashishb/patch-1

Corrected a step for local operationalization

PRMerger10 committed:
Merge pull request #30668 from carolinacmoravia/patch-530

L10088: Change comment format to HTML
